smt/pkg/smt/smt_batch.go

```diff
@@ -24,77 +24,64 @@ func NewInsertBatchConfig(ctx context.Context, logPrefix string, shouldPrintProg
 	}
 }
 
-func (s *SMT) InsertBatch(cfg InsertBatchConfig, nodeKeys []*utils.NodeKey, nodeValues []*utils.NodeValue8, nodeValuesHashes []*[4]uint64, rootNodeHash *utils.NodeKey) (*SMTResponse, error) {
-	s.clearUpMutex.Lock()
-	defer s.clearUpMutex.Unlock()
-
-	var maxInsertingNodePathLevel = 0
-	var size int = len(nodeKeys)
-	var err error
-	var smtBatchNodeRoot *smtBatchNode
-	nodeHashesForDelete := make(map[uint64]map[uint64]map[uint64]map[uint64]*utils.NodeKey)
-
-	var progressChanPre chan uint64
-	var stopProgressPrinterPre func()
-	if cfg.shouldPrintProgress {
-		progressChanPre, stopProgressPrinterPre = zk.ProgressPrinter(fmt.Sprintf("[%s] SMT incremental progress (pre-process)", cfg.logPrefix), uint64(4), false)
+func getProgressPrinterPre(logPrefix string, progressType string, size uint64, shouldPrintProgress bool) (progressChanPre *chan uint64, stopProgressPrinterPre func()) {
```

Comment:
@V-Staykov 

the code looks good,

but could you improve the readbility by adding newlines too many parameters

func name(
1 something,
2 something,
3 something,

)  (what returns1, what reutnrs2 )  {}

if you could make this change would be a great thing to maintain ( sorry if this sounds picky ) not a necessity!